.circleci/config.yml

```diff
@@ -150,8 +169,8 @@ commands:
       - restore_cache:
           name: Restore .cargo
           keys:
-            - rust-cargo-<< pipeline.parameters.cache_version >>-build-<< parameters.os >>-{{ checksum "Cargo.lock" }}
-            - rust-cargo-<< pipeline.parameters.cache_version >>-build-<< parameters.os >>
+            - rust-cargo-<< pipeline.parameters.cache_version >>-build-<< parameters.os >>-{{ arch }}-{{ checksum "Cargo.lock" }}
+            - rust-cargo-<< pipeline.parameters.cache_version >>-build-<< parameters.os >>-{{ arch }}
```

Comment:
@garypen In retrospect, I actually think that all these changes that introduce `{{arch}}` in this commit may turn out be unnecessary since this PR introduces the `_arm` designator to the `os` value further down in this file within this PR.